sources/packages/api/src/services/restriction/constants.ts

```diff
@@ -9,3 +9,24 @@ export const RESTRICTION_FEDERAL_MESSAGE =
  */
 export const RESTRICTION_PROVINCIAL_MESSAGE =
   "Your account has a hold on it. Please contact SABC to resolve.";
+/**
+ * Restriction code that are used in API.
+ * * The Restriction table will have more restriction
+ * * code, other than below mentioned codes.
+ */
+export enum RestrictionCode {
```

Comment:
I would say that we have been creating the enum more into a `model` file than in a constant file.

Reply:
the reason why I created this enum in const is it's not sure as a type of restriction code in the restriction table, it is only used  for  business logics

Reply:
As we talked about, it can be created at the service level because it is not used at the entity level.